doc/15-troubleshooting.md

```diff
@@ -176,6 +176,62 @@ C:\> cd C:\ProgramData\icinga2\var\log\icinga2
 C:\ProgramData\icinga2\var\log\icinga2> Get-Content .\debug.log -tail 10 -wait
 ```
 
+### Enable/Disable Debug Output on the fly <a id="troubleshooting-enable-disable-debug-output-api"></a>
+
+Every feature is just an Icinga 2 config object similar to Host and Service, e.g.
+the `debuglog` feature is a [FileLogger](09-object-types.md#objecttype-filelogger).
+Those can also be [managed via API](12-icinga2-api.md#icinga2-api-config-objects)
+at runtime. This is a good alternative to `icinga2 feature enable debuglog`:
+
+* Object creation/deletion via API happens immediately and requires no restart
+* Hence, the debug log is enabled exactly as long as desired
```

Comment:
```suggestion
at runtime.

This is a good alternative to `icinga2 feature enable debuglog` as object
creation/deletion via API happens immediately and requires no restart.
```

I would just continue with text, as the list makes it harder to read. Furthermore, the second bullet point adds no real value, imo.

Reply:
> the second bullet point adds no real value, imo.

Well, if your reload takes 1/2h, your debug log is enabled for 1/2h longer than necessary, spamming the disk. And if your setup is that large, it will spam a lot.

Reply:
This totally makes sense. Could you then please add this information to the documentation, because for me this was not the obvious intent of the second sentence.

Btw, I am in favor of this PR and come back to it occasionally, for example, just this morning.

Reply:
As stated above, the bullet list is hard to read. The second point is obsolete with the just introduced paragraph below. Please remove it.